libfabric.spec

```diff
@@ -49,7 +42,7 @@ BuildRequires: autoconf, automake, libtool
 %global configopts --enable-sockets --enable-verbs --enable-usnic --disable-static --disable-efa --without-gdrcopy --disable-opx --enable-tcp
```

Comment:
should we explicitly --disable-psm2 here? i dont know from top of my head what is the default now for it

Reply:
> should we explicitly --disable-psm2 here? i dont know from top of my head what is the default now for it

Without the library present (which this PR accomplishes), there is no way that the build could use it, so it would be superfluous to add `--disable-psm2` here IMO.